prime-router/src/main/kotlin/fhirengine/utils/HL7Reader.kt

```diff
@@ -350,6 +151,7 @@ class HL7Reader(private val actionLogger: ActionLogger) : Logging {
         }
 
         // map of HL7 message profiles: maps profile to configuration directory path
+        @Deprecated("This field is only in use for the CLI", level = DeprecationLevel.WARNING)
```

Comment:
Is this something that needs to be looked into? If so, can we create a story to address this and add it to the PR description? It looks like there's other stuff in here that could probably get deleted that I'm not clear why it exists anymore, like: oidProfileMap (this is leftover from the old strategy iirc)

Reply:
If it wasn't in the CLI (and dealing with that out of scope for this ticket) I would have went ahead and removed it, but I did want to keep other people from adding more stuff that uses it. I can add a ticket, yeah.